JS2-week2/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Prototype</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <form id="quiz-form">
+      <label>Question</label>
+      <input type="text" name="question" />
+
+      <label>Answers</label>
+      <input type="text" name="answers" />
+      <button type="button" onclick="submitQuestion()">Submit Quiz</button>
+    </form>
+
+    <hr />
+
+    <div id="questions-wrapper"></div>
+
```

Comment:
You should have more inputs for anwsers - I would say mininium 2 anwsers makes sense, then there is something to choose from :) 
It could be an interesting option to have min. 2 inputs for anwsers and be able to add more inputs if user would like to have more anwsers to choose from. The option to choose a correct one is also missing. 

You could add anwsers with just one input but then you need to modify your code a bit - have an anwsers array that you will push the value from input every time user pushes some `Add answer` button. Then you submit the question with all the awnsters from the array and clean the array on submit.